pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1128,7 +1128,9 @@ static ZERO_PAGE: Bytes = Bytes::from_static(&[0u8; BLCKSZ as usize]);
 //
 // Checkpoint:
 // 03 00000000 00000000 00000000 00   00000001
-
+//
+// PG_VERSION:
+// 03 00000000 00000000 00000000 00   00000002
```

Comment:
I don't see any code to create this key.

Reply:
fixed